prov/rxm/src/rxm_ep.c

```diff
@@ -451,7 +450,6 @@ static ssize_t rxm_ep_send_common(struct fid_ep *ep_fid, const struct iovec *iov
 	ret = rxm_get_conn(rxm_ep, dest_addr, &rxm_conn);
 	if (ret)
 		return ret;
-
 	if (freestack_isempty(rxm_ep->txe_fs)) {
 		FI_DBG(&rxm_prov, FI_LOG_CQ, "Exhaused tx_entry freestack\n");
 		return -FI_ENOMEM;
```

Comment:
Please drop all changes to the RXM provider from this series.  If changes are needed, please submit them separately, so they don't get lost in a large patch.